collector/cluster_health.go

```diff
@@ -77,6 +80,10 @@ func NewClusterHealth(logger log.Logger, client *http.Client, url *url.URL) *Clu
 			Name: prometheus.BuildFQName(namespace, subsystem, "total_scrapes"),
 			Help: "Current total ElasticSearch cluster health scrapes.",
 		}),
+		scrapeDuration: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: prometheus.BuildFQName(namespace, subsystem, "scrape_duration_seconds"),
```

Comment:
Hmm, maybe we should flip these per-subsystem metrics around and make them global, and use a label.

Maybe we do this in a refactor before cutting a new release? Although, we should not merge this until we cut a "safe harbor" release.